### PR TITLE
ChangeLog: add the v0.4.0 release notes

### PR DESCRIPTION
**Pull request by philips**
This doesn't include everything but the highlights of the PRs and issues
closed.

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+	* Go package added for types
+	https://godoc.org/github.com/opencontainers/image-spec/specs-go/v1 (#172)
+	* oci-image-tool: man page docs added
+	https://github.com/opencontainers/image-spec/tree/master/cmd/oci-image-tool/man
+	(#180)
+
+	Minor fixes and documentation:
+
+	* oci-image-tool: replace colon with hyphen to be windows friendly (#165)
```
Comment (wking) [inline ChangeLog]: This PR was closed, not merged.  That fix happened in #177.

**Comment by philips** [comment]
cc @opencontainers/image-spec-maintainers

**Comment by wking** [comment]
7431439 looks good to me.

**Comment by vbatts** [comment]
LGTM

**Comment by stevvooe** [comment]
LGTM

**Comment by brendandburns** [comment]
LGTM.

**Comment by jonboulle** [comment]
LGTM (for content, formatting seems a bit weird)

**Comment by philips** [comment]
@jonboulle following the weird formatting of the runtime-spec. We can reformat them elsewhere.

Also, a reminder to @brendandburns and @stevvooe an LGTM here doesn't count as a vote for the release: https://groups.google.com/a/opencontainers.org/d/msg/dev/XwIb5czltSE/GKs-6FkGCgAJ

This ChangeLog won't be merged until that thread gets LGTM from a couple more @opencontainers/image-spec-maintainers 

Thank you!

**Comment by vbatts** [comment]
which will be interesting for tagging, as a couple things have landed on master since the vote post.

**Comment by wking** [comment]
On Tue, Aug 16, 2016 at 08:12:20AM -0700, Vincent Batts wrote:
> which will be interesting for tagging, as a couple things have
> landed on master since the vote post.

Technically, the vote is to tag 2bae0dc [1], although my guess is that
everyone intends to tag 74314395 from this PR instead.  I expect the
history will end up looking something like this:

  *   e735f26 (HEAD, master) Merge remote-tracking branch 'origin/pr/195'
  |\  
  | * 7431439 (tag: v0.4.0, origin/pr/195) ChangeLog: add the v0.4.0 release notes
  * |   85d1eeb (origin/master, origin/HEAD) Merge pull request #198 from vbatts/add_README
  |\ \  
  | * | 8045bdd (origin/pr/198) oci-image-tool: add README
  |/ /  
  * |   ca97691 Merge pull request #197 from coolljt0725/fix_typo_in_descriptor
  |\ \  
  | |/  
  |/|   
  | * d400f10 (origin/pr/197) Typo: fix typo in descriptor.go
  |/  
  *   2bae0dc Merge pull request #171 from stevvooe/opaque-whiteout

With #197 and #198 being part of 0.5.0.

[1]: https://groups.google.com/a/opencontainers.org/forum/#!topic/dev/XwIb5czltSE

**Comment by caniszczyk** [comment]
Looks like there's 5 LGTMs for the release (implicit LGTM from @vbatts for calling the vote)

That meets the 2/3 bar

**Comment by philips** [comment]
Vote passed. Email sent to the mailing list. Merging!

**Comment by caniszczyk** [comment]
@philips thanks, do you plan to tag the release? https://github.com/opencontainers/image-spec/tags

**Comment by philips** [comment]
@caniszczyk @vbatts did it https://github.com/opencontainers/image-spec/releases/tag/v0.4.0

**Review by Ghostman2591 — APPROVED** [review]

**Pull request merged**